### PR TITLE
Use .npmignore to reduce distributed node package

### PR DESCRIPTION
**Pull request by dangdennis**
## Description

Late night brainstorming about becoming a more empathetic developer. We need SDKs! And we need more open source! We can potentially use this repo (or a sep one) to create our SDKs. Sorry, getting ahead of myself. 

The newly added npmignore reduces the size of our uploaded package.
1. Makes new installs quicker between package updates
2. Improves our CI size cuz of said package updates
3. A practice step towards building better OSS.

See the new version at `1.3.10-slim.0`
Screen Shot 2021-08-21 at 2 00 36 AM

## Testing

> Teach your teammates how to run and check that your work does what you say it does.

- [ ] Is QA required?

## Related issues

> Help your teammates keep track of work. Link relevant PRs and tickets.
>
> You can link Linear tickets by ID within the title or add the entire Linear ticket URL inside the description. GH integrations will link tickets and PRs.

### CODE REVIEW

**Review by xer0x — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@canopyinc/api-docs",
-  "version": "1.3.9",
+  "version": "1.3.10-slim.0",
```
Comment (xer0x) [inline package.json:3]: what a fancy version!

**Comment by xer0x** [comment]
What does the address state stuff do?

**Comment by dangdennis** [comment]
> What does the address state stuff do?

It was all generated code. We must've not generated the code after we removed the enum in the response schema

**Comment by xer0x** [comment]
> > What does the address state stuff do?
> 
> It was all generated code. We must've not generated the code after we removed the enum in the response schema

Oh, it sure did leave a bunch of junk behind.  🧹

**Pull request merged**